code/_compile_options.dm

```diff
@@ -63,11 +63,11 @@
 #endif
 
 //Update this whenever you need to take advantage of more recent byond features
-#define MIN_COMPILER_VERSION 510
+#define MIN_COMPILER_VERSION 511
 #if DM_VERSION < MIN_COMPILER_VERSION
 //Don't forget to update this part
 #error Your version of BYOND is too out-of-date to compile this project. Go to byond.com/download and update.
-#error You need version 510 or higher
+#error You need version 511 or higher

```

Comment:
If you make the server use a beta version, you MUST specify this in the error message as they will go download the stable version and get confused.